### PR TITLE
chore: add missed attributes into che-machine-exec:latest of che-theia editor

### PR DESCRIPTION
**Pull request by svor**
Signed-off-by: Valerii Svydenko <vsvydenk@redhat.com>

### What does this PR do?
Adds missed attributes into che-machine-exec component of che-theia:latest editor

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]

File: che-editors.yaml [hunk 1]
```diff
@@ -287,10 +287,16 @@ editors:
...
+          memoryLimit: 128Mi
+          memoryRequest: 32Mi
+          cpuLimit: 500m
+          cpuRequest: 30m
           command:
             - /go/bin/che-machine-exec
             - '--url'
             - '127.0.0.1:3333'
+            - '--idle-timeout'
+            - '15m'
```
Comment (l0rd) [inline che-editors.yaml:299]: If we set idling via the command line then we won't be able to specify it via the `CheCluster` (parmeters have precedence over the env variable) no?

Comment (ibuziuk) [reply che-editors.yaml:299]: well, I believe we set it already for theia for 3.0 and nothing is going to change for 3.1 I think
for 3.2 we can remove those parameters once all the dedicated idling PRs are merged.

Comment (l0rd) [reply che-editors.yaml:299]: I mean, when I opened [the idling issue](https://github.com/eclipse/che/issues/21390) I mentioned that 

> Theia definition should not specify the --idle-timeout parameter for machine exec.

Now we can temporarily set it for `stable` for consistency but as soon as @dkwon17 PRs get merged we should remove it for both `next` and `stable`.

Comment (dkwon17) [reply che-editors.yaml:299]: Yes, in the machine-exec idling PR: https://github.com/eclipse-che/che-machine-exec/pull/208, the parameters have precedence over the environment variable: https://github.com/eclipse-che/che-machine-exec/blob/aa76e6080e188822ab9c60abce6c01791befc3c7/cfg/cfg.go#L87-L97

Comment (azatsarynnyy) [inline che-editors.yaml:299]: As I remember, during today's standup call we discussed increasing it to 30 min.
no?

**Review by azatsarynnyy — APPROVED** [review]

File: che-editors.yaml (hunk 1 above)
Comment (dkwon17) [reply che-editors.yaml:299]: It has been increased to 30 minutes from the che-operator side in this PR: https://github.com/eclipse-che/che-operator/pull/1443

as defaults here: https://github.com/eclipse-che/che-operator/blob/fdbbcdb47e811a54e8e3672ba856d43a3aa5a3e0/api/v2/checluster_types.go#L96-L97

Comment (svor) [reply che-editors.yaml:299]: so, does these changes make any sense or it's better to wait when David's PRs will be merged?

**Comment by svor** [comment]
As we discussed in the daily call, I'm going to merge current PR and when @dkwon17 finishes his work on [che-machine-exec](https://github.com/eclipse-che/che-machine-exec/pull/208) side, we can cleanup che-machine-exec attributes in che-plugin registry.

@ibuziuk about back-porting these changes into 7.50. If we do that, we'll need to do another release of che-plugin-registry for 7.50. I think it's good enough to have it only in 7.51  FDYT?

**Comment by ibuziuk** [comment]
@svor wondering if we would have the idle timeout attribute for all relevant editors in downstream / 3.1 ? 
7.51.0 should be released soon, so I'm ok not to backport it if we would have --idle-timeout correctly set for downstream

**Pull request merged**